pkg/crypto/didcreator/didcreator.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package didcreator
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/crypto/internal/walletapi"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+)
+
+// DIDCreator provides features to create and query DID document
+type DIDCreator interface {
+	// Creates new DID document.
```

Comment:
prefer to start comment with function name... not sure why linter doesn't catch this

Reply:
Maybe it didn't complain when I just refactored out code without changing it.

I'll fix.

Reply:
linter doesn't catch this error for some reason ... we have the same erroneous comments in other interfaces as well.

Reply:
done